interpreter/functions/functions.go

```diff
@@ -45,6 +45,9 @@ type Overload struct {
 	// Function defines the overload with a FunctionOp implementation. May be
 	// nil.
 	Function FunctionOp
+
+	// AllowError specifies whether the Overload will tolerate error arguments.
+	AllowError bool
```

Comment:
Let's prefer a boolean `NonStrict` flag here. For the use case you described I know that only permitting errors would be desirable, but from a CEL perspective functions are either strict or non-strict and the non-strict ones permit both error and unknown values as inputs. 

From an implementation perspective, it means that you could either return an `Unknown` value when it'd provided as input, or you can treat it as an error, or whatever you like. Let me know if this makes sense to you. Thanks for your patience on the review feedback.

Reply:
Yeah, that works for me.